server/docker-entrypoint.sh

```diff
@@ -73,6 +73,7 @@ else
     DB_NAME="embedded H2"
 fi
 
+export JDBC_PARAMS=$(echo ${JDBC_PARAMS} | sed '/^$/! s/^/?/')
```

Comment:
Can you elaborate on what this is doing?

Reply:
I've commented in 1a80ce01a25a57489ef789f9d2609b84ecebf633

Reply:
Thanks. My sed skills are a bit rusty, so couldn't quite figure out what it did ;)